numpy/lib/tests/test_function_base.py

```diff
@@ -1447,6 +1448,39 @@ def test_size_zero_output(self):
         with assert_raises_regex(ValueError, 'new output dimensions'):
             f(x)
 
+    @pytest.mark.skipif(not HAS_REFCOUNT, reason="Python lacks refcounts")
+    def test_leaks(self):
+        # gh-11867
+        # np.frompyfunc(<class method>) creates a reference cycle
+        # that requires a gc collection cycle to break (on CPython 3)
+        import gc
+        gc.disable()
+        iters = 20
+        try:
+            class A(object):
+                def class_method(self, *args):
```

Comment:
This name is pretty misleading - it's not a `@classmethod`

Reply:
done